.claude/commands/integration_example.py

```
@@ -0,0 +1,127 @@
+        "/test src/",                           # Basic test
+        "/debug /test src/",                    # Debug test  
+        "/paranoid /test integration/",         # Paranoid test
+        "/debug /paranoid /test $PROJECT_ROOT/",     # Combined modes
+        "/minimal /test unit/"                  # Minimal test
+    ]
+    
+    print("=== Testing Simple Command Composition Integration ===")
+    
+    for command in test_cases:
+        print(f"\n--- Command: {command} ---")
+        claude_cli_parser_enhanced(command)
```

Ruff (0.15.5)

[error] Undefined name `claude_cli_parser_enhanced`

(F821)